app/lib/activitypub/adapter.rb

```diff
@@ -9,6 +9,7 @@ class ActivityPub::Adapter < ActiveModelSerializers::Adapter::Base
       {
         'manuallyApprovesFollowers' => 'as:manuallyApprovesFollowers',
         'sensitive'                 => 'as:sensitive',
+        'movedTo'                   => 'as:movedTo',
```

Comment:
Should you really use ActivityStreams' namespace for that? I couldn't find any documentation for “movedTo”

Reply:
I made it up since it doesn't exist, but probably should exist. @cwebber is okay with it

Reply:
Alright, that's fine with me.